src/grt/src/GlobalRouter.cpp

```diff
@@ -99,7 +99,7 @@ GlobalRouter::GlobalRouter()
       verbose_(false),
       min_layer_for_clock_(-1),
       max_layer_for_clock_(-2),
-      critical_nets_percentage_(0),
+      critical_nets_percentage_(10),
```

Comment:
@maliberty Should we set the 10% of critical nets as default in the tool code or enable it only in ORFS?

Reply:
I think we should change the default as other flows are unlikely to update and this looks to be a good change.